src/utils/analytics/events.ts

```diff
@@ -0,0 +1,18 @@
+export enum HotspotEvents {
+  ADD_GATEWAY_INITIATED = 'AddGateway Initiated',
+  ADD_GATEWAY_SUBMITTED = 'AddGateway Submitted',
+  ASSERT_LOCATION_INITIATED = 'AssertLocation Initiated',
```

Comment:
What's the difference between "initiated" and "started"?

Reply:
Essentially they are same. Let me standardize them.